index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```

Comment:
tab

Reply:
done

Reply:
?

Reply:
auto reformatting messed this up, but if fixed it.

Reply:
Did you change the configturation of the reformatting?

Reply:
yes